docs/static/breaking-changes-90.asciidoc

```diff
@@ -223,3 +223,14 @@ removed and their replacements.
 |=======================================================================
 
 ====
+
+[discrete]
+[[enterprise_search-deprecated-9.0]]
+===== App Search and Workplace Search plugins are deprecated
```

Comment:
@karenzone Maybe we should use "removed" instead of "deprecated" to indicate the plugins will not come with Logstash 9